docs/src/development/variables.md

```diff
@@ -2,115 +2,160 @@
 title: "Variables"
 weight: 5
 description: |
-  Platform.sh allows a high degree of control over both the build process and the runtime environment of a project.  Part of that control comes in the form of *variables* that are set independently of the project's code base, but available either at build or runtime for your code to use.
+  Use variables to take control over your project's build process and its runtime environment. They are set independently of the project's code base but are available to your code at build or runtime.
```

Comment:
I like your changes. 👍 One minor thing I'd try to do is to merge this paragraph with the sentence below for better flow. 

Variables allow you to control your project’s build process and its runtime environment providing your app with additional information, such as database credentials, host and port it can use, and which server to connect to. Variables are set independently of the project code base but are available to your code at build or runtime.

Reply:
Ordinarily, I think that makes some sense. But given the use of `description` (in the meta tag for SEO and on listing pages), I think it's best for descriptions to be short. Adding that other sentence might help for keywords, but would make it rather long.

If you agree, I'd add a note about descriptions to the style PR and keep this one separate.